### PR TITLE
Add node and numpy-mp openfaas templates

### PR DESCRIPTION
**Pull request by qzheng527**
Signed-off-by: Qi Zheng <qi.zheng@intel.com>

### CODE REVIEW

File: FaaS/OpenFaaS/template/numpy-mp/function/set-num-threads.sh [hunk 1]
```diff
@@ -0,0 +1,57 @@
+#!/bin/sh
```
Comment (ZhongbaoShi) [inline FaaS/OpenFaaS/template/numpy-mp/function/set-num-threads.sh]: Is this function folder under numpy-mp different meaning?

File: FaaS/OpenFaaS/template/README.md [hunk 2]
```diff
@@ -39,86 +39,18 @@ Steps could refer to
...
 
-    `echo "openblas" >> hello-dockerfile/bundles.txt`
-    `echo "wget" >> hello-dockerfile/bundles.txt`
-4.  Replace the "cat" in the Dockerfile line below per your requirement.
-    ENV fprocess="cat"
-5.  `faas-cli up -f hello-dockerfile.yml`
+* [node-clearlinux](./node-clearlinux/README.md)
 
-    Then you can invoke your python function by OpenFaas UI or faas-cli.
+* [numpy-mp](./numpy-mp/README.md)
```
[the remark this answers is not in the document]
Comment (qzheng527) [reply FaaS/OpenFaaS/template/README.md]: The numpy-mp template is not based on the published numpy-mp container.
It is based on the python container.

File: FaaS/OpenFaaS/template/go-clearlinux/README.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
+Based on clearlinux/golang container image
+
+### go-clearlinux
+1.  mkdir test && cd test
+2.  `git clone https://gitlab.devtools.intel.com/cle-dns/clearlinux-openfaas.git`
```
[the remark this answers is not in the document]
Comment (qzheng527) [reply FaaS/OpenFaaS/template/go-clearlinux/README.md]: Will fix

File: FaaS/OpenFaaS/template/python3-clearlinux/README.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
+Based on clearlinux/python:3 container image.
+Current version is 3.8.
+
+### python3-clearlinux
+1.  mkdir test && cd test
+2.  `git clone https://gitlab.devtools.intel.com/zhengq/clearlinux-openfaas.git`
```
[the remark this answers is not in the document]
Comment (qzheng527) [reply FaaS/OpenFaaS/template/python3-clearlinux/README.md]: Will fix

File: FaaS/OpenFaaS/template/node-clearlinux/README.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+1.  mkdir test && cd test
+2.  `git clone https://gitlab.devtools.intel.com/zhengq/clearlinux-openfaas.git`
+3.  cp -r clearlinux-openfaas/template .
+4.  `faas-cli new --lang dockerfile-clearlinux hello-openfaas --prefix="<your-docker-username-here>"`
+
+    Files tree as below.
+>
+    ├── hello-openfaas
+    │   ├── handler.js
+    │   └── package.json
```
[the remark this answers is not in the document]
Comment (qzheng527) [reply FaaS/OpenFaaS/template/node-clearlinux/README.md]: We don't need requirements.txt which is used by python.
For bundles.txt, it depends how much customization you need on the template.
Usually language template should be enough for the development on the template, no need extra apps install. For official language template, also no apps customization provided.

File: FaaS/OpenFaaS/template/node-clearlinux/README.md [hunk 6]
```diff
@@ -0,0 +1,24 @@
+Based on clearlinux/node:12 container image.
+
+### node-clearlinux
+1.  mkdir test && cd test
+2.  `git clone https://gitlab.devtools.intel.com/zhengq/clearlinux-openfaas.git`
```
[the remark this answers is not in the document]
Comment (qzheng527) [reply FaaS/OpenFaaS/template/node-clearlinux/README.md]: Will fix

File: FaaS/OpenFaaS/template/go-clearlinux/README.md [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+3.  cp -r clearlinux-openfaas/template .
+4.  `faas-cli new --lang go-clearlinux go-openfaas --prefix="<your-docker-username-here>"`
+
+    Files tree as below.
+>
+    ├── go-openfaas
+    │   ├── handler.go
```
[the remark this answers is not in the document]
Comment (qzheng527) [reply FaaS/OpenFaaS/template/go-clearlinux/README.md]: We don't need requirements.txt which is used by python.
For bundles.txt, it depends how much customization you need on the template.
Usually language template should be enough for the development on the template, no need extra apps install. For official language template, also no apps customization provided.

File: FaaS/OpenFaaS/template/numpy-mp/README.md [hunk 8]
```diff
@@ -0,0 +1,43 @@
+Based on clearlinux/python:3 container image.
+Current version is 3.8.
+It can set OMP_NUM_THREADS dynamically at runtime to accelerate
+performance according to assigned computing resources.
+
+### numpy-clearlinux
+1.  mkdir test && cd test
+2.  `git clone https://gitlab.devtools.intel.com/zhengq/clearlinux-openfaas.git`
```
[the remark this answers is not in the document]
Comment (qzheng527) [reply FaaS/OpenFaaS/template/numpy-mp/README.md]: Will fix

File: FaaS/OpenFaaS/template/numpy-mp/function/set-num-threads.sh (hunk 1 above)
Comment (qzheng527) [reply FaaS/OpenFaaS/template/numpy-mp/function/set-num-threads.sh]: no special meaning

File: FaaS/OpenFaaS/template/go-clearlinux/README.md (hunk 7 above)
[the remark this answers is not in the document]
Comment (qzheng527) [reply FaaS/OpenFaaS/template/go-clearlinux/README.md]: I will add bundles.txt as another commit

File: FaaS/OpenFaaS/template/node-clearlinux/README.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (qzheng527) [reply FaaS/OpenFaaS/template/node-clearlinux/README.md]: I will add bundles.txt as another commit

**Pull request merged**